.idea/misc.xml

```diff
@@ -3,6 +3,9 @@
   <component name="DesignSurface">
     <option name="filePathToZoomLevelMap">
       <map>
+        <entry key="..\:/Users/Julie/Documents/Documents/University/UofA/Year 2 A/CMPUT 301/repos/InterestingRespectfulProfessional/app/src/main/res/layout/add_habit_dialog_box.xml" value="0.2546875" />
+        <entry key="..\:/Users/Julie/Documents/Documents/University/UofA/Year 2 A/CMPUT 301/repos/InterestingRespectfulProfessional/app/src/main/res/layout/add_habit_event_dialog.xml" value="0.268" />
+        <entry key="..\:/Users/Julie/Documents/Documents/University/UofA/Year 2 A/CMPUT 301/repos/InterestingRespectfulProfessional/app/src/main/res/layout/fragment_habit_event_list.xml" value="0.2546875" />
```

Comment:
These shouldn't be here, as this path is local to your machine.

Reply:
- [ ] remove local path